apps/backup

```diff
@@ -64,11 +64,15 @@ B2NAND() {
 	if [ "$1" != "-b" ]; then
 		if [ ! -f $NAND/$MUVERSION-RCC_fs0.bin ]; then
 			if [[ $SIZE -gt 150000 ]]; then
-				on -f mmx cp -r $BACKUPFOLDER/$MUVERSION-RCC_fs0.bin $NAND/ 2>> $LOG
-				if [[ "$TRAINVERSION" != *MHI2Q* ]]; then
-					on -f mmx cp -r $BACKUPFOLDER/$MUVERSION-MMX_fs0.bin $NAND/ 2>> $LOG
+				if [ -f $BACKUPFOLDER/$MUVERSION-RCC_fs0.bin ]; then
```

Comment:
that should never happen, as RCC is checked and created just before as part of the backup

Reply:
It happens because at installation only basic backup is done.